h/views/api/helpers/media_types.py

```diff
@@ -31,13 +31,23 @@ def valid_media_types(versions=None):
     * An Accept header containing at least one of the media types returned by
       this function.
 
-    :arg versions: List of version strings to include in the valid media types.
-                   Defaults to the app's known version list.
+    :rtype: list(str)
+    """
+    valid_types = ["*/*", "application/json"] + version_media_types()
+    return valid_types
+
+
+def version_media_types(versions=None):
```

Comment:
Do you envision this helper being used outside of this module? Currently it looks like it is just a helper for `valid_media_type`.